src/main/java/cz/fi/muni/pa165/Model/Entity/Car.java

```diff
@@ -0,0 +1,127 @@
+package cz.fi.muni.pa165.Model.Entity;
+
+import cz.fi.muni.pa165.Model.CarState;
+
+import javax.persistence.*;
+import javax.persistence.Id;
+import javax.validation.constraints.NotNull;
+import java.util.Date;
+import java.util.UUID;
+
+/**
+ * Created by SARMIR on 26. 10. 2016.
+ */
+
+@SuppressWarnings("WeakerAccess")
+@Entity
+public class Car {
```

Comment:
pri `User` tu mame package viditelnost, asi aj tu sa hodi, aby to nebolo uplne public, ci?